tests/functional/ui/test_health_overview.py

```
@@ -600,3 +701,128 @@ def test_health_score_with_blackbox_and_metrics_exporter_shutdown(
+                constants.ALERT_ODF_NODE_LATENCY_HIGH_OSD_NODES,
+                "custom-odf-latency-rule.yaml",
+                threading_lock,
+            )
+            unique_alert_types = [constants.ALERT_ODF_NODE_LATENCY_HIGH_OSD_NODES]
+            logger.info(f"Using mocked alert: {unique_alert_types}")
+
+        alerts_to_disable = []
+        self.update_alert_map(threading_lock)
+        for alert in unique_alert_types:
+            if ALERT_MAP[alert] > 0:
+                alerts_to_disable.append(alert)
```

Ruff (0.15.15)

[warning] Use a list comprehension to create a transformed list

(PERF401)